lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -145,7 +146,12 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :versions_file_path,
                                        description: "Path to the VERSIONS.md file",
                                        optional: true,
-                                       type: String)
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :is_prerelease,
```

Comment:
The parameter everywhere is called `include_prerelease`. Should we rename the parameter to match?

Reply:
I was torn because this isn't about including prereleases but saying this bump _is_ a prerelease 🤷‍♂️ So I _think_ this one is okay to stay different? 🤔 Thoughts?